shell/platform/embedder/tests/embedder_test_context.cc

```diff
@@ -13,40 +14,12 @@
 namespace flutter {
 namespace testing {
 
+static constexpr const char* kAppAOTELFFileName = "app_elf_snapshot.so";
```

Comment:
Isn't the `const` here not redundant with `constexpr`?

Reply:
I wrote that on autopilot and was debating if it was redundant before you convinced me offline that that it was indeed unnecessary. When I changed it to remove the const though `ISO C++11 does not allow conversion from string literal to 'char *const'`.